src/newton_raphson.toit

```diff
@@ -94,12 +125,18 @@ solve --initial/num=0.0 --goal/num=0.0 --max-iterations=20 --precision=1e9 [--fu
       return old-diff.abs < new-diff.abs ? x : new-x
     previous = x
     x = new-x
-  no_convergence.call x
-  return x
+  return if-not-converge.call x
 
 /**
-Variant of $(solve --initial --goal --max-iterations --precision [--function] [--derivative] [--no_convergence]).
+Variant of $(solve --initial --goal --max-iterations --precision [--function] [--derivative] [--if-not-converge]).
```

Comment:
Wouldn't `on-not-converge` be more in line with how we typically name such blocks?

Reply:
I would say 'on-x' should be used if the callback can be invoked multiple times, or if there is a "callback".
In this case, it feels more like the `if-absent` to me.